Formula/ganglia.rb

```diff
@@ -19,7 +19,7 @@ class Ganglia < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "apr"
+  depends_on "apr-util"
```

Comment:
There's no need to change this since it was already fixed. `"apr"` is valid. The audit check refers to `:apr`.